openmc/mgxs/library.py

```diff
@@ -1184,13 +1176,11 @@ def check_library_for_openmc_mgxs(self):
         # Ensure absorption is present
         if 'absorption' not in self.mgxs_types:
             error_flag = True
-            msg = '"absorption" MGXS type is required but not provided.'
-            warn(msg)
+            warn('"absorption" MGXS type is required but not provided.')
```

Comment:
My OCD self is coming out, but how about opening this (and the following similar) error messages with "A"?